storage/ndb/src/kernel/blocks/dbtc/Dbtc.hpp

```diff
@@ -2698,6 +2698,8 @@ class Dbtc
 private:
   Uint32 c_time_track_histogram_boundary[TIME_TRACK_HISTOGRAM_RANGES];
   bool c_time_track_activated;
+
+  Uint32 c_use_query_thread_for_locked_reads;
```

Comment:
Perhaps it should be initialized in `void Dbtc::initData()` too, not just when reading config?